### ISSUE TITLE
Issue with numeric schema name when adding to sources.yml

### ISSUE THREAD

**Issue by nikomataaa** [fishtown-analytics/dbt#2630]
### Describe the bug
I found yesterday that DBT couldn't accept the number type of schema name when I added to `sources.yml`, since it wanted to have it as String instead. I found similar issue was already resolved on this PR already : https://github.com/fishtown-analytics/dbt/pull/2208

### Steps To Reproduce
So for example I have to add this to `sources.yml` : 
```
  - name: ga360_session
    database: ga360-prod
    schema: 196936242
    tables:
      - name: ga_sessions_2020*
```
Since the schema name is `196936242` and it's number, DBT doesn't want that, it wants the String instead

### Expected behavior
Since it was already solved on this [PR](https://github.com/fishtown-analytics/dbt/pull/2208), so I expect that any input is converted to string instead

### Screenshots and log output
<img width="1055" alt="Screen Shot 2020-07-16 at 08 40 58" src="https://user-images.githubusercontent.com/55773604/87616839-3e94e080-c740-11ea-8eab-ca35ea67d185.png">

### System information
**Which database are you using dbt with?**
- [ ] postgres
- [ ] redshift
- [x] bigquery
- [ ] snowflake
- [ ] other (specify: ____________)

**The output of `dbt --version`:**
```
installed version: 0.17.0
   latest version: 0.17.0

Up to date!

Plugins:
  - bigquery: 0.17.0
  - snowflake: 0.17.0
  - redshift: 0.17.0
  - postgres: 0.17.0
```

**The operating system you're using:**
MacOS Catalina 10.15.5

**The output of `python --version`:**
Python 3.7.4

**Comment by jtcohen6** [comment]
Hey @nikomataaa, I believe this will be resolved by a change we're making in v0.17.1 to roll back native-env rendering by default ([discourse](https://discourse.getdbt.com/t/coming-soon-v0-17-1/1406)).

Your code above worked locally for me by:
* Wrapping the numeric value in single quotes, i.e. `schema: '196936242'`
* Running with the release candidate `v0.17.1-rc4`

Could you give that a go, and let me know if it works for you as well?

**Comment by nikomataaa** [comment]
@jtcohen6 aah I see, I'll try it with my team and let you know afterwards !

**Comment by jtcohen6** [comment]
Closing because I believe this issue to be resolved in v0.17.1. Feel free to re-open if you're still running into trouble.
